api-rest/src/main/java/com/example/controller/StudentController.java

```diff
@@ -0,0 +1,36 @@
+package com.example.controller;
+
+import com.example.domain.Student;
+import com.example.dto.StudentDTO;
+import com.example.mapper.StudentMapperAPI;
+import com.example.usecase.student.StudentCreateUseCase;
+import com.example.usecase.student.StudentUseCase;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class StudentController {
+
+    private final StudentUseCase studentUseCase;
+
+    private final StudentCreateUseCase studentCreateUseCase;
+
+    private final StudentMapperAPI studentMapperStructAPI;
+
+
+    @GetMapping("/student")
```

Comment:
pls add mapping /v1/student, u need to use version in all mappings

Reply:
as i understand i need to create package with version mapper DO, done